code/modules/surgery/surgery_step.dm

```diff
@@ -12,10 +12,6 @@
 
 /datum/surgery_step/proc/try_op(mob/user, mob/living/target, target_zone, obj/item/tool, datum/surgery/surgery, try_to_fail = FALSE)
 	var/success = FALSE
-	if(surgery.organ_to_manipulate && !target.getorganslot(surgery.organ_to_manipulate))
```

Comment:
These lines 15-18 are a basic check in tg surgery and have nothing to do with pain.